CreeDictionary/API/apps.py

```diff
@@ -74,15 +74,9 @@ def initialize_affix_search():
     logger.info("Building tries for affix search...")
     from .models import Wordform
 
-    cree_letter_to_ascii = {
-        ascii_letter: ascii_letter for ascii_letter in string.ascii_lowercase
-    }
-    cree_letter_to_ascii.update(
-        {"â": "a", "ā": "a", "ê": "e", "ē": "e", "ī": "i", "î": "i", "ô": "o", "ō": "o"}
-    )
```

Comment:
we already have a `remove_cree_diacritics` function and this here does the same thing.

So I extracted a function here